base/client.jl

```diff
@@ -393,10 +393,10 @@ function _start()
                 term = Terminals.TTYTerminal(get(ENV, "TERM", @static Sys.iswindows() ? "" : "dumb"), STDIN, STDOUT, STDERR)
                 global is_interactive = true
                 color_set || (global have_color = Terminals.hascolor(term))
-                quiet || REPL.banner(term,term)
+                banner && REPL.banner(term,term)
                 if term.term_type == "dumb"
                     active_repl = REPL.BasicREPL(term)
-                    quiet || warn("Terminal not fully functional")
+                    banner && warn("Terminal not fully functional")
```

Comment:
`quiet` could be (in the future) an umbrella option for a bag of others, including `banner` and supressing this warning above; but until there are more candidates to be controlled by `quiet`, it seems fine to have banner control this warning.